client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/screen-main.tsx

```diff
@@ -100,47 +108,57 @@ const ScreenMain = ( {
 				<HStack direction="column" alignment="top" spacing="4">
 					<NavigatorItemGroup title={ translate( 'Layout' ) }>
 						<NavigationButtonAsItem
+							checklist
+							checked={ hasHeader }
 							path={ NAVIGATOR_PATHS.HEADER }
 							icon={ header }
 							aria-label={ translate( 'Header' ) }
 							onClick={ () => onSelect( 'header' ) }
 						>
-							<span className="pattern-layout__list-item-text">{ translate( 'Header' ) }</span>
```

Comment:
The class `.pattern-layout__list-item-text` is only used in the screen-section in the component `PatternLayout` so I'm removing it from the here (the main-screen) and adding a class `.navigator-button__text` in NavigatorButton. cc: @arthur791004